include/solvers/postprocessing_cfd.h

```diff
@@ -273,6 +273,23 @@ calculate_flow_rate(const DoFHandler<dim> &    dof_handler,
                     const Mapping<dim> &       mapping);
 
 
+/**
+ * @brief Calculates the minimum and maximum temperature in the simulation domain
+ *
+ * @param dof_handler. The argument used to get finite elements
+ *
+ * @param face_quadrature_formula The face quadrature formula for the calculation
```

Comment:
This is not a face quadrature right? please review these comments. I think there is some copy paste errors there :)!

Reply:
True :) corrected.